tools/bin/goreleaser_utils

```diff
@@ -70,6 +69,19 @@ get_remote_plugin_paths() {
     full_plugin_path=$(go list -m -f "{{.Dir}}" "$plugin_dep_name")"$plugin_main"
     echo "$full_plugin_path"
   done
+
+  plugins=(
+    "github.com/smartcontractkit/chainlink-cosmos@a8ef07bd9cd050cc29938292395603f3ccec787c|/pkg/cosmos/cmd/chainlink-cosmos"
+    "github.com/smartcontractkit/chainlink-starknet/relayer@47ab7749ab3e4d7e0ce3910a1a1fbf71640637b0|/pkg/chainlink/cmd/chainlink-starknet"
+  )
+
+  for plugin in "${plugins[@]}"; do
+    plugin_dep_name=$(echo "$plugin" | cut -d"|" -f1)
+    plugin_main=$(echo "$plugin" | cut -d"|" -f2)
+
+    full_plugin_path=$(go mod download -json "$plugin_dep_name" | jq -r .Dir)"$plugin_main"
+    echo "$full_plugin_path"
+  done
```

Comment:
There is probably a better way to dedupe this code since the only difference is inside of the `$()`